devicetypes/smartthings/zigbee-power-meter.src/zigbee-power-meter.groovy

```diff
@@ -104,6 +105,36 @@ def parse(String description) {
                         map.name = "energy"
                         map.value = zigbee.convertHexToInt(it.value)/(energyDivisor * 1000)
                         map.unit = "kWh"
+                        
+                        if (isEZEX()) {
+                            def currentEnergy = zigbee.convertHexToInt(it.value) / 1000
+                            def prevPowerConsumption = device.currentState("powerConsumption")?.value
+                            Map previousMap = prevPowerConsumption ? new groovy.json.JsonSlurper().parseText(prevPowerConsumption) : [:]
```

Comment:
you need to import `groovy.json.JsonSlurper` at the top of the file or I expect this to throw an error.

Reply:
Actually, it has not caused error yet. Also, in the DTH, zigbee-metering-plug-power-consumption-report, there is no import statement for groovy.json.JsonSlurper either. 

Even though, do you want me to add "import groovy.json.JsonSlurper"?

Reply:
> you need to import `groovy.json.JsonSlurper` at the top of the file or I expect this to throw an error.

I've added import statement cause you haven't answered. please check it asap.